primitives/api/proc-macro/src/decl_runtime_apis.rs

```diff
@@ -380,6 +380,7 @@ fn generate_call_api_at_calls(decl: &ItemTrait) -> Result<TokenStream> {
 		// Generate the generator function
 		result.push(quote!(
 			#[cfg(any(feature = "std", test))]
+			#[allow(clippy::too_many_arguments)]
```

Comment:
currently we don't use clippy to check our code, so we don't specify those allowance. maybe we should start, but then better to open an issue to get people opinion IMHO.
```suggestion
```

Reply:
ah my bad I didn't see it was inside macro generated code, then I'm ok to keep it if substrate user like clippy it makes sense to have it.